Simplenote/MagicLinkInvalidView.swift

```diff
@@ -0,0 +1,92 @@
+import Foundation
+import SwiftUI
+import Gridicons
+
+
+// MARK: - MagicLinkConfirmationView
+//
+struct MagicLinkInvalidView: View {
```

Comment:
I think there is some duplication here with the invalid view and the check your email view.  They are the same except that the invalid view has the new link button.  Not sure if it is worth it, but I wonder if there is a way to save on the duplication here